package.json

```diff
@@ -29,24 +29,22 @@
     "tslib": "^2.3.0"
```

Comment:
**Suggestion** could we bump this one?

Reply:
Bumped up to 2.4.0 in this commit: 020928374c41ed2bfe0c8fa785eb1ed410457ccf

Let me know if you would like it at a newer version!